src/main/java/com/icthh/xm/tmf/ms/prepaybalance/web/v4/V4BucketDelegate.java

```diff
@@ -24,4 +24,13 @@
     public ResponseEntity<List<Bucket>> listBucket(String fields, Integer offset, Integer limit) {
         return ResponseEntity.ok().build();
     }
+
+    @Timed
+    @Override
+    @LogicExtensionPoint(value = "RetrieveBucket", resolver = ProfileKeyResolver.class)
+    @PrivilegeDescription("Privilege to retrieve bucket")
+    @PreAuthorize("hasPermission({'profile': @headerRequestExtractor.profile}, 'RETRIEVE.BUCKET')")
```

Comment:
can u pls add service name prefix to permission key, like: BALANCE.RETRIEVE.BUCKET (I`d also switch retrieve and bucket). The same for list bucket permission